prow/plugins.yaml

```diff
@@ -11,6 +11,7 @@ triggers:
   - tensorflow/k8s
   - GoogleCloudPlatform/k8s-multicluster-ingress
   trusted_org: kubernetes
+  join_org_url: "https://github.com/kubernetes/community/blob/master/community-membership.md#member"
```

Comment:
community nit: can we use `https://git.k8s.io/community/community-membership.md#member` for k8s, please?